app/src/main/java/com/infomaniak/mail/ui/main/user/ManageMailAddressViewModel.kt

```diff
@@ -43,7 +43,7 @@ class ManageMailAddressViewModel(application: Application) : AndroidViewModel(ap
     }
 
     private fun updateMailboxes(user: User) = viewModelScope.launch(Dispatchers.IO) {
-        val mailboxes = ApiRepository.getMailboxes(user.id).data ?: return@launch
+        val mailboxes = ApiRepository.getMailboxes(AccountUtils.getHttpClient(user.id)).data ?: return@launch
```

Comment:
getMailboxes needed a signature change to be used in Login Activity because of the noInterceptor client. This is the only place where the optional parameter was given, so it needed refactor.

Reply:
Yes I still don't see the relation.

Because it is only called by `observeAccounts`. And so it is only with the `currentUser`.